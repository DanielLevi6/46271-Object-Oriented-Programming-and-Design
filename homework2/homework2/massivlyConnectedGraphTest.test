# massivlyConnectedGraphTest test file

# Create a graph
CreateGraph myGraph

# Add nodes
CreateNode node1 1
CreateNode node2 2
CreateNode node3 2
CreateNode node4 3
CreateNode node5 2
CreateNode node6 4
CreateNode node7 5
CreateNode node8 6
CreateNode node9 4
CreateNode node10 3
CreateNode node11 2
CreateNode node12 4
CreateNode node13 1
CreateNode node14 1

AddNode myGraph node1
AddNode myGraph node2
AddNode myGraph node3
AddNode myGraph node4
AddNode myGraph node5
AddNode myGraph node6
AddNode myGraph node7
AddNode myGraph node8
AddNode myGraph node9
AddNode myGraph node10
AddNode myGraph node11
AddNode myGraph node12
AddNode myGraph node13
AddNode myGraph node14

# Add an edge
AddEdge myGraph node1 node1
AddEdge myGraph node1 node2
AddEdge myGraph node1 node3
AddEdge myGraph node1 node4
AddEdge myGraph node1 node5
AddEdge myGraph node1 node6
AddEdge myGraph node1 node7
AddEdge myGraph node1 node8
AddEdge myGraph node1 node9
AddEdge myGraph node1 node10
AddEdge myGraph node1 node11
AddEdge myGraph node1 node12
AddEdge myGraph node1 node13
AddEdge myGraph node2 node2
AddEdge myGraph node2 node3
AddEdge myGraph node2 node4
AddEdge myGraph node2 node5
AddEdge myGraph node2 node6
AddEdge myGraph node2 node7
AddEdge myGraph node2 node8
AddEdge myGraph node2 node9
AddEdge myGraph node2 node10
AddEdge myGraph node2 node11
AddEdge myGraph node2 node12
AddEdge myGraph node2 node13
AddEdge myGraph node2 node14
AddEdge myGraph node3 node3
AddEdge myGraph node3 node4
AddEdge myGraph node3 node5
AddEdge myGraph node3 node6
AddEdge myGraph node3 node7
AddEdge myGraph node3 node8
AddEdge myGraph node3 node9
AddEdge myGraph node3 node10
AddEdge myGraph node3 node11
AddEdge myGraph node3 node12
AddEdge myGraph node3 node13
AddEdge myGraph node3 node14
AddEdge myGraph node4 node4
AddEdge myGraph node4 node5
AddEdge myGraph node4 node6
AddEdge myGraph node4 node7
AddEdge myGraph node4 node8
AddEdge myGraph node4 node9
AddEdge myGraph node4 node10
AddEdge myGraph node4 node11
AddEdge myGraph node4 node12
AddEdge myGraph node4 node13
AddEdge myGraph node4 node14
AddEdge myGraph node5 node5
AddEdge myGraph node5 node6
AddEdge myGraph node5 node7
AddEdge myGraph node5 node8
AddEdge myGraph node5 node9
AddEdge myGraph node5 node10
AddEdge myGraph node5 node11
AddEdge myGraph node5 node12
AddEdge myGraph node5 node13
AddEdge myGraph node5 node14
AddEdge myGraph node6 node6
AddEdge myGraph node6 node7
AddEdge myGraph node6 node8
AddEdge myGraph node6 node9
AddEdge myGraph node6 node10
AddEdge myGraph node6 node11
AddEdge myGraph node6 node12
AddEdge myGraph node6 node13
AddEdge myGraph node6 node14
AddEdge myGraph node7 node7
AddEdge myGraph node7 node8
AddEdge myGraph node7 node9
AddEdge myGraph node7 node10
AddEdge myGraph node7 node11
AddEdge myGraph node7 node12
AddEdge myGraph node7 node13
AddEdge myGraph node7 node14
AddEdge myGraph node8 node8
AddEdge myGraph node8 node9
AddEdge myGraph node8 node10
AddEdge myGraph node8 node11
AddEdge myGraph node8 node12
AddEdge myGraph node8 node13
AddEdge myGraph node8 node14
AddEdge myGraph node9 node9
AddEdge myGraph node9 node10
AddEdge myGraph node9 node11
AddEdge myGraph node9 node12
AddEdge myGraph node9 node13
AddEdge myGraph node9 node14
AddEdge myGraph node10 node10
AddEdge myGraph node10 node11
AddEdge myGraph node10 node12
AddEdge myGraph node10 node13
AddEdge myGraph node10 node14
AddEdge myGraph node11 node11
AddEdge myGraph node11 node12
AddEdge myGraph node11 node13
AddEdge myGraph node11 node14
AddEdge myGraph node12 node12
AddEdge myGraph node12 node13
AddEdge myGraph node12 node14
AddEdge myGraph node13 node13
AddEdge myGraph node13 node14
AddEdge myGraph node14 node14

# Print some stuff
ListNodes myGraph
ListChildren myGraph node1
ListChildren myGraph node2
ListChildren myGraph node3
ListChildren myGraph node4
ListChildren myGraph node5
ListChildren myGraph node6
ListChildren myGraph node7
ListChildren myGraph node8
ListChildren myGraph node9
ListChildren myGraph node10
ListChildren myGraph node11
ListChildren myGraph node12
ListChildren myGraph node13
ListChildren myGraph node14

# Find a path
FindPath myGraph node1 -> node14 