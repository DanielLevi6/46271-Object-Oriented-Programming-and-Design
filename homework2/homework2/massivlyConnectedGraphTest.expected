# massivlyConnectedGraphTest test file

# Create a graph
created graph myGraph

# Add nodes
created node node1 with cost 1
created node node2 with cost 2
created node node3 with cost 2
created node node4 with cost 3
created node node5 with cost 2
created node node6 with cost 4
created node node7 with cost 5
created node node8 with cost 6
created node node9 with cost 4
created node node10 with cost 3
created node node11 with cost 2
created node node12 with cost 4
created node node13 with cost 1
created node node14 with cost 1

added node node1 to myGraph
added node node2 to myGraph
added node node3 to myGraph
added node node4 to myGraph
added node node5 to myGraph
added node node6 to myGraph
added node node7 to myGraph
added node node8 to myGraph
added node node9 to myGraph
added node node10 to myGraph
added node node11 to myGraph
added node node12 to myGraph
added node node13 to myGraph
added node node14 to myGraph

# Add an edge
added edge from node1 to node1 in myGraph
added edge from node1 to node2 in myGraph
added edge from node1 to node3 in myGraph
added edge from node1 to node4 in myGraph
added edge from node1 to node5 in myGraph
added edge from node1 to node6 in myGraph
added edge from node1 to node7 in myGraph
added edge from node1 to node8 in myGraph
added edge from node1 to node9 in myGraph
added edge from node1 to node10 in myGraph
added edge from node1 to node11 in myGraph
added edge from node1 to node12 in myGraph
added edge from node1 to node13 in myGraph
added edge from node2 to node2 in myGraph
added edge from node2 to node3 in myGraph
added edge from node2 to node4 in myGraph
added edge from node2 to node5 in myGraph
added edge from node2 to node6 in myGraph
added edge from node2 to node7 in myGraph
added edge from node2 to node8 in myGraph
added edge from node2 to node9 in myGraph
added edge from node2 to node10 in myGraph
added edge from node2 to node11 in myGraph
added edge from node2 to node12 in myGraph
added edge from node2 to node13 in myGraph
added edge from node2 to node14 in myGraph
added edge from node3 to node3 in myGraph
added edge from node3 to node4 in myGraph
added edge from node3 to node5 in myGraph
added edge from node3 to node6 in myGraph
added edge from node3 to node7 in myGraph
added edge from node3 to node8 in myGraph
added edge from node3 to node9 in myGraph
added edge from node3 to node10 in myGraph
added edge from node3 to node11 in myGraph
added edge from node3 to node12 in myGraph
added edge from node3 to node13 in myGraph
added edge from node3 to node14 in myGraph
added edge from node4 to node4 in myGraph
added edge from node4 to node5 in myGraph
added edge from node4 to node6 in myGraph
added edge from node4 to node7 in myGraph
added edge from node4 to node8 in myGraph
added edge from node4 to node9 in myGraph
added edge from node4 to node10 in myGraph
added edge from node4 to node11 in myGraph
added edge from node4 to node12 in myGraph
added edge from node4 to node13 in myGraph
added edge from node4 to node14 in myGraph
added edge from node5 to node5 in myGraph
added edge from node5 to node6 in myGraph
added edge from node5 to node7 in myGraph
added edge from node5 to node8 in myGraph
added edge from node5 to node9 in myGraph
added edge from node5 to node10 in myGraph
added edge from node5 to node11 in myGraph
added edge from node5 to node12 in myGraph
added edge from node5 to node13 in myGraph
added edge from node5 to node14 in myGraph
added edge from node6 to node6 in myGraph
added edge from node6 to node7 in myGraph
added edge from node6 to node8 in myGraph
added edge from node6 to node9 in myGraph
added edge from node6 to node10 in myGraph
added edge from node6 to node11 in myGraph
added edge from node6 to node12 in myGraph
added edge from node6 to node13 in myGraph
added edge from node6 to node14 in myGraph
added edge from node7 to node7 in myGraph
added edge from node7 to node8 in myGraph
added edge from node7 to node9 in myGraph
added edge from node7 to node10 in myGraph
added edge from node7 to node11 in myGraph
added edge from node7 to node12 in myGraph
added edge from node7 to node13 in myGraph
added edge from node7 to node14 in myGraph
added edge from node8 to node8 in myGraph
added edge from node8 to node9 in myGraph
added edge from node8 to node10 in myGraph
added edge from node8 to node11 in myGraph
added edge from node8 to node12 in myGraph
added edge from node8 to node13 in myGraph
added edge from node8 to node14 in myGraph
added edge from node9 to node9 in myGraph
added edge from node9 to node10 in myGraph
added edge from node9 to node11 in myGraph
added edge from node9 to node12 in myGraph
added edge from node9 to node13 in myGraph
added edge from node9 to node14 in myGraph
added edge from node10 to node10 in myGraph
added edge from node10 to node11 in myGraph
added edge from node10 to node12 in myGraph
added edge from node10 to node13 in myGraph
added edge from node10 to node14 in myGraph
added edge from node11 to node11 in myGraph
added edge from node11 to node12 in myGraph
added edge from node11 to node13 in myGraph
added edge from node11 to node14 in myGraph
added edge from node12 to node12 in myGraph
added edge from node12 to node13 in myGraph
added edge from node12 to node14 in myGraph
added edge from node13 to node13 in myGraph
added edge from node13 to node14 in myGraph
added edge from node14 to node14 in myGraph

# Print some stuff
myGraph contains: node1 node10 node11 node12 node13 node14 node2 node3 node4 node5 node6 node7 node8 node9
the children of node1 in myGraph are: node1 node10 node11 node12 node13 node2 node3 node4 node5 node6 node7 node8 node9
the children of node2 in myGraph are: node10 node11 node12 node13 node14 node2 node3 node4 node5 node6 node7 node8 node9
the children of node3 in myGraph are: node10 node11 node12 node13 node14 node3 node4 node5 node6 node7 node8 node9
the children of node4 in myGraph are: node10 node11 node12 node13 node14 node4 node5 node6 node7 node8 node9
the children of node5 in myGraph are: node10 node11 node12 node13 node14 node5 node6 node7 node8 node9
the children of node6 in myGraph are: node10 node11 node12 node13 node14 node6 node7 node8 node9
the children of node7 in myGraph are: node10 node11 node12 node13 node14 node7 node8 node9
the children of node8 in myGraph are: node10 node11 node12 node13 node14 node8 node9
the children of node9 in myGraph are: node10 node11 node12 node13 node14 node9
the children of node10 in myGraph are: node10 node11 node12 node13 node14
the children of node11 in myGraph are: node11 node12 node13 node14
the children of node12 in myGraph are: node12 node13 node14
the children of node13 in myGraph are: node13 node14
the children of node14 in myGraph are: node14

# Find a path
shortest path in myGraph: node1 node13 node14
